.github/workflows/ubuntu.yml

```diff
@@ -77,6 +75,9 @@ jobs:
             env_file: actions-310-numpydev.yaml
             pattern: "not slow and not network and not single_cpu"
             test_args: "-W error::DeprecationWarning -W error::FutureWarning"
+            # TODO(cython3): Re-enable once next-beta(after beta 1) comes out
+            # There are some warnings failing the build with -werror
+            pandas_ci: 0
```

Comment:
Is there a commit available to install after beta 1 where the warnings are fixed? Setting this to 0 will affect some tests that are using `is_ci_environment()`

Reply:
I'll need to file a bug with Cython. If I did this correctly, only the numpy dev build should be affected.

Reply:
Okay yeah should be okay since it's only affecting one build, but good to have the tests as consistent as possible if cython can be bumped on this build